scripts/extract_jartic_2024_2025.py

```
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)